src/components/MonthlyGraphs/MonthlyGraphs.tsx

```diff
@@ -95,11 +96,13 @@ const MonthlyGraphs = () => {
   }
 
   return (
-    <section className="graph-container">
-      <h3 className="month-title">{monthString}</h3>
-      {!!monthlyMoods.length && <Chart type="line" data={dataset} options={options} />}
-      {!monthlyMoods.length && !error && <h2>There aren't any mood entries for this month yet!</h2>}
-      {error && <h2>Sorry, something went wrong!</h2>}
+    <section className="graph-section">
+      <div className="graph-container">
+        <h3 className="month-title">{monthString}</h3>
+        {!!monthlyMoods.length && <Chart type="line" data={dataset} options={options} />}
```

Comment:
Isn't two !! equal the same thing? lol

Reply:
Oh this is to prevent a zero (0) from rendering when there is no monthly moods. It is saying, this is just a conditional and not something to be evaluated and rendered to the dom.